onecodex/viz/_distance.py

```diff
@@ -221,7 +221,7 @@ def plot_distance(
         open_links_in_new_tab(concat_chart)
 
         if return_chart:
-            return concat_chart
+            return concat_chart, chart.data
```

Comment:
This'll break backwards compatibility for existing users of the client library. Is there a way to access the data with `concat_chart.data`?

Reply:
@jairideout Hmm..Unfortunately no, it's not available via `concat_chart`. I suppose we could add a method that is a duplicate of this method that returns `chart.data` instead, but that's a lot of duplicate code. Do you have other ideas for how best to handle it?

Reply:
I would double-check with @clausmith or @Keats on this, but we probably don't want to make a backwards incompatible change to the `plot_distance()` method because things could stop working for existing users (e.g. in notebooks or their own custom workflows).

One way to solve this is by moving all of the logic in `plot_distance()` into a "private" method (e.g. `_plot_distance()`) that returns `concat_chart` and `chart`. Then `plot_distance()` can call `_plot_distance()` and the logic won't be duplicated. On the Custom Plots side, we could use `_plot_distance()` to get the data we need for exports.

Reply:
@jairideout That sounds good to me.
@Keats @clausmith How does the above sound to you both?

Reply:
Update: I found a way to access the data, actually! So this change isn't needed anymore.